consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -202,7 +202,9 @@ public void run() {
             findRequest.where(and(
                     withValue("configurationName = " + configuration.getConfigurationName()),
                     withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
-                    not(withSubfield("jobExecutions", withValue("completedFlag = true")))));
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_SUCCESS'"))),
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_PARTIAL'")))
```

Comment:
I think the query is right now by the axioms you made.  Maybe we should also ignore the MigrationJobExecution that were aborted.

I also have a doubt if jobExecutions# would be greater than 1 or, if it is greater, if the reaming items would be processed as the run() method would called just once by the ConsistencyChecker on https://github.com/luan-cestari/lightblue-migrator/blob/8955b9c69e23d4cf17cc3187ed700d497ee2f87c/consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java#L153-153 . In other words, my concern is if all MigrationJobExecution from a MigrationJob are being executed as expected (which I guess not) and if we are processing all MigrationJobExecution , would this query also include the MigrationJobs that at least one of MigrationJobExecution.jobStatus is not Completed ?